source/simulator/parameters.cc

```diff
@@ -338,6 +338,18 @@ namespace aspect
                            "The maximum number of line search iterations allowed. If the "
                            "criterion is not reached after this iteration, we apply the scaled "
                            "increment to the solution and continue.");
+
+        prm.declare_entry ("Use Newton residual scaling method", "false",
+                           Patterns::Bool (),
+                           "This method allows to slowly introduce the derivatives based on the improvement "
+                           "of the residual. If we do not use it, we just set it so the newton_derivative_scaling_factor "
+                           "goes from zero to one when switching on the Newton solver.");
+
+        prm.declare_entry ("Minimum linear Stokes solver tolerance", "0.9",
```

Comment:
I feel like this needs to be called `Maximum linear Stokes solver tolerance` since technically the value is not allowed to be higher. I agree however that using `Maximum` and `Minimum` for solver tolerances is confusing.